### ISSUE TITLE
[Breakpoints] in sourcemapped files are ignored

### ISSUE THREAD

**Issue by TitanNano** [devtools-html/debugger.html#3756]
Breakpoints set in source-mapped files are ignored. When I set the breakpoints in the generated source bundle the debugger halts. When I then step through the lines, the debugger jumps between the generated source and the mapped original source files, for every other line. 

This is not the first time this bug appeared, it comes and goes every few weeks.

**Comment by jasonLaster** [comment]
Thanks for the report @TitanNano. Could you try to reproduce the bug with one of our two test apps:

http://aws-sensorweb-static-site.s3-website-us-west-2.amazonaws.com/
http://firefox-dev.tools/debugger-examples/examples/increment/

ideally increment because it's smaller :)

**Comment by TitanNano** [comment]
I can't reproduce it with either of the apps. I think it's a source-mapping issue. Is there any way I can check if there is anything wrong with our source-maps? 

When I set the breakpoints in the generated source, the debugger jumps to the correct mapped line. but when I set a break point in a mapped source mostly the debugger does not halt there. So some part of the debugger is not happy with our source map. 

Also as I said, when stepping though the lines it constantly jumps between the mapped and the generated source.
